Tests/ClientRuntimeTests/NetworkingTests/URLSession/FoundationStreamBridgeTests.swift

```diff
@@ -60,7 +60,7 @@ class FoundationStreamBridgeTests: XCTestCase {
 
             // Create a stream bridge with our original data & open it
             let bufferedStream = BufferedStream(data: originalData, isClosed: true)
-            let subject = FoundationStreamBridge(readableStream: bufferedStream, bufferSize: bufferSize)
+            let subject = FoundationStreamBridge(readableStream: bufferedStream, bufferSize: bufferSize, logger: TestLogger())
```

Comment:
Like the stream bridge itself, this test file is copied verbatim from `main` branch.